lua/wire/server/wirelib.lua

```diff
@@ -1180,6 +1181,18 @@ function WireLib.IsValidMaterial(material)
 	return material
 end
 
+function WireLib.SetColor(ent, color)
```

Comment:
This won't work correctly, we need to leave the rendermode alone unless it's `RENDERMODE_NORMAL` or `RENDERMODE_TRANSALPHA`:

```Lua
Prop:setRenderMode(_RENDERMODE_GLOW)
Prop:setColor(255, 255, 255, 64)
```

Reply:
Ok, I can fix that